j2v/utils/helpers.py

```diff
@@ -33,22 +32,21 @@ def is_str_timestamp(dim_val):
     :return: True only if string represents a timestamp
     """
     try:
-        if datetime.datetime.strptime(dim_val, "%Y-%m-%dT%H:%M:%S.%fZ"):
+        if datetime.strptime(dim_val, "%Y-%m-%dT%H:%M:%S.%fZ").isoformat():
             return True
     except:
         return False
 
 
-def is_unix_timestamp(dim_name, dim_val):
+def is_unix_timestamp(dim_val):
     """
-       Checks if a string represents a timestamp
-       :param dim_name:
+       Checks if dim_val represents a timestamp
        :param dim_val:
        :return: True only if string represents a timestamp
        """
-    digits = len(str(dim_val))
-    possible_digits = {10, 13, 16}
-    return dim_val > 0 and "time" in dim_name.lower() and digits in possible_digits
+    date_now = datetime.now()
+    time_diff = timedelta(days=365)
+    return dim_val > 0 and date_now + time_diff > datetime.fromtimestamp(dim_val) > date_now - time_diff
```

Comment:
You don't need `dime_val > 0`.

Reply:
Yeah I'll change it!

Reply:
We do need this check as negative time is still parsed!